quantum/snippets.js

```diff
@@ -30,7 +30,9 @@ const QUANTUM_REGISTER =
 const BARRIER =
 `qc.barrier(%s)`;
 
-const HADAMARD_GATE = 
+const CNOT_GATE = `qc.cx(s%, s%)`;
```

Comment:
Looks great, thank you! Good job Zhiqiang, I am very happy with all the files you modified!

One small thing: here you have written 's%' instead of '%s' !